cluster/ssl_config.go

```diff
@@ -28,38 +28,39 @@ import (
 )
 
 // SSLConfig is SSL configuration for client.
-// SSLConfig has tls.Config embedded in it so that users can set any field
-// of tls config as they wish. SSL config also has some helpers such as SetCaPath, AddClientCertAndKeyPath to
-// make configuration easier for users.
+// SSLConfig has tls.Config embedded in it so that users can set any field of tls config as they wish.
 type SSLConfig struct {
```

Comment:
Should we change the name of this to TLSConfig as I see from the standard library of Golang, they don't even implement the now deprecated SSLv3, etc.

Reply:
I don't think there are many serious applications that use SSL, but it became something like an umbrella term which covers TLS too. It's also `SSLConfig` in the Java client, so keeping it that way seems better to me.